src/collection.js

```diff
@@ -764,18 +764,31 @@ export default class Collection {
    * - {Collection.strategy} strategy: See {@link Collection.strategy}.
    * - {Boolean} ignoreBackoff: Force synchronization even if server is currently
    *   backed off.
+   * - {String} remote The remote Kinto server endpoint to use (default: null).
    *
    * @param  {Object} options Options.
    * @return {Promise}
+   * @throws {Error} If an invalid remote option is passed.
    */
-  sync(options={strategy: Collection.strategy.MANUAL, headers: {}, ignoreBackoff: false}) {
+  sync(options={
+    strategy: Collection.strategy.MANUAL,
+    headers: {},
+    ignoreBackoff: false,
+    remote: null,
+  }) {
+    const previousRemote = this.api.remote;
+    if (options.remote) {
+      console.log("using custom remote", options.remote);
```

Comment:
Should kinto.js contain console.log statements? Just noting that this seems to be the first/only console.log statement so far.